taf/TAF_Runner.py

```diff
@@ -1057,6 +1057,15 @@ def repack_code(self, code: str, str_num: str, padzero: bool = False, num_length
 
             return repack
 
+    # Private function to make sure runtime param is forced to boolean.
+    # Python evaluates everything to True except 0 and True
+    # so we're forcing it to False otherwise for safety reasons
+    def __forceBool__(self, _boolean_: int = 0):
+        if str(_boolean_) not in ('0', '1'):
+            self.logger.warning(f"'run_stats_only' parameter is passed as '{_boolean_}' but converted to False. If you meant 'True' please pass '1'")
```

Comment:
I'd opt for a raised exception if the parameter is invalid and keep it to "Please pass 0 or 1". I think this statement would be easy to ignore in the event of an unexpected value since execution would continue.

Reply:
Fair enough.